lib/redis_memo/memoize_query/cached_select.rb

```diff
@@ -157,6 +160,11 @@ def map_substitutes(values)
     end
   end
 
+  # Extract bind params from the query by inspecting the SQL's AST recursively
```

Comment:
Can we add that the actual bind params get put in a thread local variable, and this method returns a boolean to indicate whether the query should be cached